modules/javafx.graphics/src/main/java/com/sun/javafx/css/CompoundSelector.java

```diff
@@ -78,20 +69,21 @@ public List<SimpleSelector> getSelectors() {
     }
 
     private final List<Combinator> relationships;
-    // /**
-    //  * The relationships between the selectors
-    //  * @return Immutable List&lt;Combinator&gt;
-    //  */
-    // public List<Combinator> getRelationships() {
-    //     return relationships;
-    // }
+
+    /**
+     * The relationships between the selectors
+     * @return Immutable List&lt;Combinator&gt;
```

Comment:
minor: would {@code List<Combinator>} be a better choice here, if it works?

Reply:
Fixed